templates/mail/wait-list.html

```diff
@@ -0,0 +1,70 @@
+{% extends 'base.html' %} {% block body %}
+<h1>Join the waitlist</h1>
+<div id="mcWaitlistForm">
+  <!-- Begin Mailchimp Waitlist Form -->
+  <div id="mc_embed_shell">
+    <div id="mc_embed_signup">
+      <form
+        action="{{form_action}}"
+        method="post"
+        id="mc-embedded-subscribe-form"
+        name="mc-embedded-subscribe-form"
+        class="validate"
+        target="_self"
+        novalidate=""
+      >
+        <div id="mc_embed_signup_scroll">
+          <div class="indicates-required">
+            <span class="asterisk">*</span> indicates required
+          </div>
+          <div class="mc-field-group">
+            <label for="mce-EMAIL"
+              >Email Address <span class="asterisk">*</span></label
+            ><input
+              type="email"
+              name="EMAIL"
+              class="required email"
+              id="mce-EMAIL"
+              required=""
+              value=""
+            />
+          </div>
+          <div hidden="">
+            <input type="hidden" name="tags" value="10526643" />
+          </div>
+          <div id="mce-responses" class="clear">
+            <div
+              class="response"
+              id="mce-error-response"
+              style="display: none"
+            ></div>
+            <div
+              class="response"
+              id="mce-success-response"
+              style="display: none"
+            ></div>
+          </div>
+          <div style="position: absolute; left: -5000px" aria-hidden="true">
+            <input
+              type="text"
+              name="b_f8051cc8637cf3ff79661f382_a508104cf8"
+              tabindex="-1"
+              value=""
+            />
+          </div>
+          <div class="clear">
+            <input
+              type="submit"
+              name="subscribe"
+              id="mc-embedded-subscribe"
+              class="button"
+              value="Subscribe"
+            />
+          </div>
+        </div>
+      </form>
+    </div>
+  </div>
+  <!--End mc_embed_waitlist-->
+</div>
```

Comment:
The rest of this form is from their "copy this html" blerb right?

Reply:
That's correct - it's embed code that Natalie sent over